### ISSUE TITLE
Brotli compression performance improvement

### ISSUE THREAD

**Issue by SachaMorard** [edgee-cloud/edgee#40]
**Is your feature request related to a problem? Please describe.**
Brotli compression is slower than other encoding algorithms 

Edgee adds a header to the response which indicates the compute duration (`x-edgee-compute-duration`)
When an upstream server responds a gzip compressed response, the compute duration on my local env is okay (~10ms)
When an upstream server responds a brotli compressed response, the compute duration on my local env is too slow (~120ms)

The slowdown seems to originate from this part:
```rust
Some("br") => { // handle brotli encoding
    // q: quality (range: 0-11), lgwin: window size (range: 10-24)
    let mut encoder  = CompressorWriter::new(Vec::new(), 4096, 11, 24);
    encoder.write_all(new_body.as_bytes())?;
    encoder.flush()?;
    encoder.into_inner()
}
```

**Describe the solution you'd like**
- Test if brotli compression settings can help
- Look at the strategies and solutions used by other proxies 
- Test other libraries

**Comment by asolimando** [comment]
Here are some figures for `x-edgee-compute-duration` on my machine with different setups: 
- no compression: ~3ms
- brotli (quality 5): ~9 ms
- brotli (quality 11): ~101 ms <-- that's the current hard-coded value
- gzip: ~9 ms

So already by lowering the quality to `5` we have a performance closer to that of `gzip`, of course a couple of runs without proper benchmarking isn't conclusive, but it already suggests that there is room for improvement by changing some of the default parameters (with #97 in place it would probably be easier to have better figures).

Regarding what other reverse proxies do:
Nginx allows to tune several brotli parameters, including [quality/compression level](https://github.com/google/ngx_brotli?tab=readme-ov-file#brotli_comp_level), [buffer size](https://github.com/google/ngx_brotli?tab=readme-ov-file#brotli_buffers), [window size](https://github.com/google/ngx_brotli?tab=readme-ov-file#brotli_window) and the [minimum payload size](https://github.com/google/ngx_brotli?tab=readme-ov-file#brotli_min_length).

Additionally, not all MIME types equally benefit from compression, so we have an additional parameter for the [MIME types](https://github.com/google/ngx_brotli?tab=readme-ov-file#brotli_types) for which one wants brotli encoding to be activated.

In summary, a reasonable starting point is to lower the compression quality parameter, and possibly add all or some of these configuration knobs to the Edgee configuration to make it tunable.
